finalPrep/annotations.js

```diff
@@ -17,26 +20,35 @@ function buildABear(name, age, fur, clothes, specialPower) {
 
   return builtBear
 }
-
+//arguments
 buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
```

Comment:
I would encourage you to be a little more specific here - lines 24 and 25 are invoking (or calling/executing) the `buildABear` function and passing in several arguments, for example.